src/webviews/main.ts

```diff
@@ -342,7 +348,7 @@ function resolveWebviewHtml(params: {
     webviewJs: string
     main: vscode.Uri
 }): string {
-    const resolvedParams = { ...params, connectSource: 'none' }
+    const resolvedParams = { ...params, connectSource: `'none'` }
```

Comment:
Code was pulled from the same commit so this minor fix for the console warning is included.